### PR TITLE
Added a dropdown to MeshModelComponent to display Component Details

### LINKED ISSUE #7620: [UI] Create a dropdown to show mesh model summary
#### Current Behavior
For now, the mesh model summary only shows component's, and relationship's name and count. 

#### Desired Behavior
We want to show JSON of the clicked component having the same **Model** and clicked relationship having the same **Kind** in a drop down menu. 

#### Implementation
Use `/api/meshmodels/models/{model}/components/{name} ` to fetch model summary and display it in a MUI dropdown component. 

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by smartdev09**
**Notes for Reviewers**

This PR fixes #7620 

How should this be tested? 

meshmodel dropdown

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
It’s nice to see movement on this, @UsmanDev09. 

These folks might have some feedback: @MdSahil-oss @theBeginner86 @Abhishek-kumar09 @meshery/maintainers

File: ui/components/MesherySettings.js [hunk 1]
```diff
@@ -351,9 +373,13 @@ class MesherySettings extends React.Component {
     ]
 
     const options = {
-      filter : false,
+      filter : true,
```
Comment (theBeginner86) [inline ui/components/MesherySettings.js:376]: Would you please turn off the filters? 
Considering UI doesn't implement filters for these data-tables.

Screenshot 2023-05-01 at 7 58 12 PM

**Review by theBeginner86 — COMMENTED** [review]
hmm, relationships also seems to be getting these dropdown. Would you please look into it?

Screenshot 2023-05-01 at 8 01 33 PM

**Review by theBeginner86 — COMMENTED** [review]
At a given instant only one dropdown is clickable. Is that intended?
> Users might get confused, if the carrot dropdown isn't disabled. Considering, this is intended.

https://user-images.githubusercontent.com/73700530/235468155-67ca97ea-3957-4360-8efc-4c2988a7eab5.mov

**Comment by theBeginner86** [comment]
> hmm, relationships also seems to be getting these dropdown. Would you please look into it?
> Screenshot 2023-05-01 at 8 01 33 PM

Fixing above should also fix before issue of multiple request to backend.
Screenshot 2023-05-01 at 8 05 05 PM

**Comment by harkiratsm** [comment]
It would be nice if a page would be dedicated just to meshmodel details, with an admin only access.  // @leecalcote @Abhishek-kumar09

**Review by theBeginner86 — COMMENTED** [review]
Good stuff @UsmanDev09  🙌  💯

This update was very much needed.

File: ui/components/MesherySettings.js (hunk 1 above)
Comment (UsmanDev09) [reply ui/components/MesherySettings.js]: Yep, I have turned off the filters

File: ui/components/MesherySettings.js [hunk 2]
```diff
@@ -274,6 +277,25 @@ class MesherySettings extends React.Component {
     }
   }
 
+
+  getComponentsFromModel = (modelName, version)  => {
```
[the remark this answers is not in the document]
Comment (UsmanDev09) [reply ui/components/MesherySettings.js:281]: Both implementations are doable whatever you think might be better.

File: ui/components/MesherySettings.js (hunk 1 above)
Comment (theBeginner86) [reply ui/components/MesherySettings.js]: Thanks

**Review by theBeginner86 — DISMISSED** [review]
LGTM

**Comment by theBeginner86** [comment]
@UsmanDev09 

If you may please consider signing-off your last few commits?
https://github.com/meshery/meshery/pull/7632/checks?check_run_id=13150194854

**Review by MUzairS15 — COMMENTED** [review]
@UsmanDev09 Thanks for creating PR, really good work here 👏🏻.

Just a small item 
Screenshot 2023-05-01 at 10 17 42 PM

The size of Meshery logo and text is not proper, will you also address this and make sure size is in accordance?

**Comment by l5io** [comment]
🚀 Preview for commit db0fc858573996ef1ea7194c4e9420898dd97770 at: https://6450bfc4ac35101d52c3ef59--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@UsmanDev09 will you resync your fork with upstream master?

**Comment by MdSahil-oss** [comment]
@UsmanDev09 Great work, Looks good to me, But can we highlight border of the opened table for some seconds? like happens in stack overflow (It would be better if we could do this).

**Comment by leecalcote** [comment]
@UsmanDev09 is this PR still valid given #7672 ?

**Comment by UsmanDev09** [comment]
@leecalcote  No, I am closing this PR

**Pull request closed without merging**